app/lib/event_store/event.rb

```diff
@@ -40,11 +40,11 @@ def load(str)
       end
     end
 
-    serialize :data, WithGlobalId
+    serialize :data, coder: WithGlobalId
     # Can't really use ActiveJob::Arguments because it does not support Time type
```

Comment:
btw this statement is maybe not true anymore. I remember this thing related to mailers where we enabled date and time serialization. But not sure. Just saying, not a request for a change.

Reply:
Hm, yeah, maybe it's good to check whether this still applies.